packages/plugins/rpc-manager/src/lib/eth_signTypedData.ts

```diff
@@ -18,13 +18,28 @@ export default class EthSignTypedData extends RpcModifier {
     // - eth_signTypedData_v3
     // - eth_signTypedData_v4
     // - personal_signTypedData (parity)
-    if (params.request.method.includes("signTypedData")) {
-      // indicate that we do not want this call to go to the node
-      params.sendToNode = false;
+    if (!params.request.method.includes("signTypedData")) {
       return callback(null, params);
     }
+
+    try {
+      const [fromAddr] = params.request.params;
+
+      const account = this.nodeAccounts.find(acc => (
+        Web3.utils.toChecksumAddress(acc) ===
+        Web3.utils.toChecksumAddress(fromAddr)
+      ));
```

Comment:
Maybe we could pull this out in to a method, since it's used in both the request and response?
```
private fromNodeAccount(request) {
  const [fromAddr] = params.request.params;
  return this.nodeAccounts.find(acc => (
    Web3.utils.toChecksumAddress(acc) ===
    Web3.utils.toChecksumAddress(fromAddr)
  ));
}
```